libraries/rush-lib/src/logic/operations/ShardedPhaseOperationPlugin.ts

```diff
@@ -10,7 +10,7 @@ import type {
 import { RushConstants } from '../RushConstants';
 import { NullOperationRunner } from './NullOperationRunner';
 import { Operation } from './Operation';
-import { normalizeNameForLogFilenameIdentifiers } from './OperationMetadataManager';
+// import { normalizeNameForLogFilenameIdentifiers } from './OperationMetadataManager';
```

Comment:
```suggestion
```

Reply:
Yeah I needed to do a onceover on this diff before I published this PR :).

I was running out the door.